KsApi/MockService.swift

```diff
@@ -200,7 +201,8 @@
         currency: currency,
         buildVersion: buildVersion,
         deviceIdentifier: deviceIdentifier,
-        perimeterXClient: perimeterXClient
+        perimeterXClient: perimeterXClient,
+        fetchProjectResponse: nil
```

Comment:
Do we have to do this for each of the GraphQL queries/mutations?

Reply:
This is still our `ServiceType` interface which will work the same way, each time we add a new query yes we'd add it to this protocol, add the query etc. Behind this protocol the Apollo client will be used to make the request.

Reply:
> Yea, I don't think we need this line as it's covered below when set our `Result`'s to `nil`.

Actually, adding this just suppresses an Xcode warning that "all calls to this function will call itself", it just differentiates it enough for that warning to be silenced.